upcoming_certificate_changes_readme.md

```diff
@@ -35,4 +35,12 @@ so users are strongly recommended to upgrade to the 2.X.X releases once they hav
 **Users who don't upgrade to the packages in [this release](https://github.com/Azure/azure-iot-sdk-java/releases/tag/lts_7_2021)
 or to the upcoming 2.X.X packages will begin experiencing unrecoverable, consistent connection failures from their devices starting June 2022.**
 
+[A preview release](https://github.com/Azure/azure-iot-sdk-java/releases/tag/2021-08-30) has been published that 
+contains a preview of the new 2.X.X clients that we plan on bringing into master in the near future. This release 
+contains many breaking changes including the changes listed above, but is substantially more future proof than the 
+1.X.X clients. Please try it out and let us know on our [discussions page](https://github.com/Azure/azure-iot-sdk-java/discussions) 
```

Comment:
nit: looks like we're already inviting participation in the discussions page a bit further down in the document, this can probably be removed